admin/server/reports.go

```diff
@@ -452,6 +452,7 @@ func (s *Server) yamlForManagedReport(opts *adminv1.ReportOptions, ownerUserID s
 	res.Annotations.AdminManaged = true
 	res.Annotations.AdminNonce = time.Now().Format(time.RFC3339Nano)
 	res.Annotations.WebOpenProjectSubpath = opts.OpenProjectSubpath
+	res.Annotations.DashboardActivePage = opts.DashboardActivePage
```

Comment:
1. Consider using `Web` prefix instead of `Dashboard` – e.g. maybe in the future, we have reports for non-dashboard pages in the UI
2. What value will this have? If it's an URL, could it be incorporated into `WebOpenProjectSubpath`?

Reply:
Ya we could say `WebActivePage`. It is sadly not part of the subpath but stored in the base64 blob. I would assume the type MetricsView / Dashboard / Chart would be based on the query somehow. If not we could add that later on and `nil` would default to `MetricsView`

Reply:
Okay what about calling it `WebShowPage` or something like that then?

Reply:
Ya that sounds good